lib/api/src/js/externals/memory.rs

```diff
@@ -179,6 +179,11 @@ impl Memory {
     pub fn duplicate(&mut self, _store: &impl AsStoreRef) -> Result<VMMemory, MemoryError> {
         self.handle.duplicate()
     }
+
+    #[allow(unused)]
+    pub fn make_inaccessible(&self, store: &impl AsStoreRef) -> Result<(), MemoryError> {
+        Ok(())
```

Comment:
Since this is not implemented for JS it should probably return an error?

Reply:
Updated it to return `MemoryError::NotImplemented` and modified the `cleanup` call to ignore this particular error (given it should not stop the runtime from cleaning up)